src/table/TableBodyWrapper.jsx

```diff
@@ -39,24 +48,33 @@ export default function TableBodyWrapper({ tableData, constraints, selectionsAPI
   return (
     <TableBody>
       {rows.map((row) => (
-        <TableRow hover role="checkbox" tabIndex={-1} key={row.key}>
+        <TableRow
+          hover={selState.isEnabled && hoverEffect}
+          role="checkbox"
+          tabIndex={-1}
+          key={row.key}
+          className={selState.isEnabled && hoverEffect && classes.hoverTableRow}
```

Comment:
I'd say we would still want the hover effect when selections are disabled

Reply:
sure